package.json

```diff
@@ -47,7 +47,7 @@
 		"immer": "^8.0.1",
 		"immutable": "3.8.2",
 		"lodash-es": "^4.17.11",
-		"marked": "^0.7.0",
+		"marked": "^4.0.10",
```

Comment:
Grote bump, dat gaat nergens anders problemen geven met mogelijke breaking changes?

Reply:
Volgens dependabot niet (#1306), ik heb het even mee in de release getrokken om uit te testen tijden feedback-verwerking